app/server/auth/authenticate.py

```diff
@@ -12,6 +12,16 @@
 from app.config.config import settings
 
 
+@router.get("/gitlab/login")
+async def login_gitlab():
+    APP_ID, REDIRECT_URI = (
+        settings.GITLAB_CLIENT_ID,
+        "http://localhost:8080/api/auth/callback/gitlab",
+    )
+    url = f"https://gitlab.com/oauth/authorize?client_id={APP_ID}&redirect_uri={REDIRECT_URI}&response_type=code"
+    return RedirectResponse(url=url)
+
+
```

Comment:
We can remove this now. I have figured a way from the FE.

Reply:
That's great boss